.github/workflows/run-tests.yml

```diff
@@ -517,10 +517,11 @@ jobs:
       - name: Run functional test
         if: ${{ inputs.run_single_functional_test != true || (inputs.run_single_functional_test == true && contains(fromJSON(needs.set-up-single-test.outputs.dbs), env.PERSISTENCE_DRIVER)) }}
         timeout-minutes: ${{ fromJSON(needs.set-up-single-test.outputs.github_timeout) }} # make sure this is larger than the test timeout in the Makefile
-        run: make OTEL=true functional-test-coverage
```

Comment:
`OTEL=true` isn't needed anymore; as the OTEL exporter is now hooked up programatically.